arm/Microsoft.ContainerRegistry/registries/deploy.bicep

```diff
@@ -97,6 +97,9 @@ param zoneRedundancy string = 'Disabled'
 @description('Optional. All replications to create.')
 param replications array = []
 
+@description('Optional. All webhooks to create')
+param webhooks array = []
```

Comment:
Please add a test case (add webhooks into a parameter file or create a new parameter file, that uses webhooks)

Reply:
check

Reply:
Please let the pipeline run on your branch and add the successful pipeline run into the PR.